### PR TITLE
🤖📯 GraphQL Schema changed

### PR DESCRIPTION
**Pull request by octokitbot**
<details>
  <summary><h3>Features (5b0f1200d146eff14c452820bd8576ed92801c52)</h3></summary>
  
  ```markdown
- ⚠️ Removed support for `Enterprise#userAccounts`
- `Organization#sponsorActivities({actions})`
- Deprecated `{ProjectNextField,ProjectNextFieldCommon}#dataType -> ProjectNextFieldType.TASKS`
- ⚠️ Removed support for `{ProjectNextField,ProjectNextFieldCommon}#dataType -> ProjectNextFieldType.TRACKS`
- `ProjectV2#{fields,items,repositories,views}({orderBy})`
- `{ProjectV2FieldCommon,ProjectV2Field,ProjectV2IterationField}#dataType -> ProjectV2FieldType.TASKS`
- ⚠️ Removed support for `{ProjectV2FieldCommon,ProjectV2Field,ProjectV2IterationField}#dataType -> ProjectV2FieldType.TRACKS`
- `ProjectV2Item#fieldValueByName`
- `ProjectV2Item#fieldValues({orderBy})`
- `ProjectV2ItemFieldPullRequestValue#pullRequests({orderBy})`
- ⚠️ Removed support for `ProjectV2View#items`
- `ProjectV2View#{orderBy,{verticalGroupBy,visibleFields}({orderBy})}`
- `PullRequestThread#{comments,id,isCollapsed,isOutdated,isResolved,pullRequest,repository,resolvedBy,viewerCanReply,viewerCanResolve,viewerCanUnresolve}`
- `Repository#vulnerabilityAlerts({dependencyScopes})`
- `RepositoryVulnerabilityAlert#dependencyScope`
- `SecurityAdvisoryPackage#ecosystem -> SecurityAdvisoryEcosystem.ACTION`
- `{Organization,User}#sponsorsActivities({actions})`
- `TreeEntity#size`
  ```
</details>

### Migrations (b22f8244abdf1c82edeb37a96b7ffb09570a47ad)
- `StartRepositoryMigrationInput#targetRepoVisibility`

<details>
  <summary><h3>Features (fdf964828969dcaa7fc0d18a6d3225fa8b3e05cd)</h3></summary>
  
  ```markdown
- `DiscussionCategory#slug`
- `Repository#discussionCategory({slug})`
  ```
</details>

### Updated schemas (ecc25ad81bf2bd82927338abc2ec4cd4d67cfa83)

<details>
  <summary><h3>Features (09170bd9b5d4ce264603a075145902e21702952b)</h3></summary>
  
  ```markdown
- add 'UNAFFILIATED' value on 'EnterpriseUserAccountMembershipRole' enum
  ```
</details>

### Updated schemas (9c3c0b54356da64160f9dd8985395b3ad8ffa6d1)

<details>
  <summary><h3>Features (2fd00362d227765452022cf3c7339a681bf11458)</h3></summary>
  
  ```markdown
- restored 'TRACKS' value on 'ProjectNextFieldType' enum
  ```
</details>

### CODE REVIEW

**Comment by oscard0m** [comment]
I created a description and updated commit messages to reflect what changes are provided per each commit.

Should we cherry-pick them somehow the changes to reduce the number of changes released in a single version? @octokit/js-community

**Comment by timrogers** [comment]
I think having a bunch of changes in one version is fine - even good! - but some of these changes look surprising. I'll do this digging to check what's up and if this is all okay.

**Comment by oscard0m** [comment]
> I'll do this digging to check what's up and if this is all okay.

Let us know what you find. I'm curious!

File: schema.graphql
Comment (timrogers) [inline schema.graphql:10359]: **⚠️ This is a breaking change**

`Enterprise.userAccounts` was deprecated in favour of `Enterprise.members`, with notice given that it would disappear around `2022-07-01`. According to our logs, there were only single-digit numbers of people still using it.

**This change is safe to merge.**

File: schema.graphql
Comment (timrogers) [inline schema.graphql:27655]: **⚠️ This is a breaking change**

The switch from "Tracks" to "Tasks" is a breaking change that is in the process of being reverted within GitHub.

**This change is not safe to merge. We should wait for it to be reverted before we merge the schema changes.**

File: schema.graphql
Comment (timrogers) [inline schema.graphql:29687]: **⚠️ This is a breaking change**

Support for listing items on a view was merged prematurely. This change has been rolled back, and we know from logs that very few (low single-digit numbers) of people were using it.

**This change is safe to merge.**

File: schema.graphql
Comment (timrogers) [inline schema.graphql:29893]: See above comment on `ProjectV2View.items`.

**This change is safe to merge.**

**Review by timrogers — CHANGES_REQUESTED** [review]
Blocking PR merge until the `Tracks` ➡️  `Tasks` change is reverted.

File: schema.graphql
Comment (timrogers) [inline schema.graphql:12712]: > feat: add support for new `UNAFFILIATED` value on `EnterpriseUserAccountMembershipRole` enum

Comment (oscard0m) [reply schema.graphql:12712]: updated commit accordingly

File: schema.graphql
Comment (oscard0m) [reply schema.graphql]: Schemas are up to date with this again: https://github.com/octokit/graphql-schema/commit/2fd00362d227765452022cf3c7339a681bf11458

**Comment by oscard0m** [comment]
> Blocking PR merge until the `Tracks` ➡️ `Tasks` change is reverted.

Are we ready to merge now, @timrogers ? See https://github.com/octokit/graphql-schema/commit/2fd00362d227765452022cf3c7339a681bf11458

**Review by timrogers — APPROVED** [review]
Looking good now ✅  How do we usually structure the release notes for this project? It looks like we have a [standard format](https://github.com/octokit/graphql-schema/releases/tag/v10.74.0), which ideally we'd keep to.

**Comment by wolfy1339** [comment]
> Looking good now :white_check_mark: How do we usually structure the release notes for this project? It looks like we have a [standard format](https://github.com/octokit/graphql-schema/releases/tag/v10.74.0), which ideally we'd keep to.

The release notes in all Octokit projects are auto-generated from the commit message. If there's many things that are added, or changed, you can go back and edit the release notes manually to add more details

**Review by timrogers — APPROVED** [review]
The latest commit just contains a description change.

**Review by timrogers — APPROVED** [review]
All looks good 🥰 Let's get this merged.

**Comment by oscard0m** [comment]
> @gr2m

Not mergeable, 'test' check is pending. @timrogers

**Comment by timrogers** [comment]
> > All looks good 🥰 Let's get this merged.
> 
> Not mergeable, 'test' check is pending. @timrogers

I think that's because another change came through 😓 When I reviewed, I'm pretty sure it was green.

**Comment by oscard0m** [comment]
> > > All looks good 🥰 Let's get this merged.
> > 
> > 
> > Not mergeable, 'test' check is pending. @timrogers
> 
> I think that's because another change came through 😓 When I reviewed, I'm pretty sure it was green.

Maybe the commit containing the 'WIP' word? Seems to be running now.

**Comment by timrogers** [comment]
You could very well be right!

**Review by timrogers — APPROVED** [review]
All of these changes are my fault 😇 I spent an hour fixing random schema bugs.

**Comment by oscard0m** [comment]
> All of these changes are my fault 😇 I spent an hour fixing random schema bugs.

No problem. Maybe we can discuss in the future how to improve this process to make it more automatic (if possible).

**Pull request merged**
